flask_meld/component.py

```diff
@@ -59,43 +58,54 @@ def load_module_from_path(full_path, module_name):
 
 
 class Component:
-    def __init__(self, id=None, request=None, **kwargs):
+    def __init__(self, id=None, form=None, **kwargs):
         if not id:
             id = uuid.uuid4()
+        self.errors = {}
+        self._form = None
         self.__dict__.update(**kwargs)
         self.id = id
-        self._form = None
-        self.errors = {}
 
         if hasattr(self, "form_class"):
             # tricky: https://flask-wtf.readthedocs.io/en/stable/api.html
             # need to pass formdata=None or flask-wtf will try to use the
             # flask request object to populate the form
             self._form = getattr(self, "form_class")(formdata=None)
-            self._set_form_data()
+            for field in self._form:
+                d = {"meld:model": field.name}
```

Comment:
I try to avoid single char variable names: 1. readability / 2. possible clash with debugger (`pdb`) commands